seatunnel-connectors-v2/connector-console/src/main/java/org/apache/seatunnel/connectors/seatunnel/console/sink/ConsoleSinkWriter.java

```diff
@@ -40,8 +42,8 @@ public class ConsoleSinkWriter extends AbstractSinkWriter<SeaTunnelRow, Void> {
     public ConsoleSinkWriter(SeaTunnelRowType seaTunnelRowType, SinkWriter.Context context) {
         this.seaTunnelRowType = seaTunnelRowType;
         this.context = context;
-        System.out.printf("fields : %s%n", StringUtils.join(seaTunnelRowType.getFieldNames(), ", "));
-        System.out.printf("types : %s%n", StringUtils.join(seaTunnelRowType.getFieldTypes(), ", "));
+        log.info("fields: {}", StringUtils.join(seaTunnelRowType.getFieldNames(), ", "));
+        log.info("types: {}", StringUtils.join(seaTunnelRowType.getFieldTypes(), ", "));
```

Comment:
The suggested output format is：`A<INT>, B<STRING>, C<BOOLEAN>,...`

Reply:
![image](https://user-images.githubusercontent.com/9366589/208338837-077a8769-b15f-4240-9d5a-48dea5fe237c.png)
All fields info has been outputted as suggested format.